src/components/modules/MainPage/MainCom/StyledMainCom.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export const SectionOne = styled.div`
+        width: "90%",
+        marginLeft: "auto",
+        marginRight: "auto",
+        marginTop: "3rem",
```

Comment:
let's remove sections as they're not part of the task

Reply:
removed